src/components/code-snippet/_code-snippet.scss

```diff
@@ -294,3 +294,326 @@
+
+  // Inline Code Snippet
+  .#{$prefix}--snippet--inline {
+    @include reset;
+    position: relative;
+    display: inline;
+    padding: 0;
+    font-size: inherit;
+    border: 1px solid transparent;
+    border-radius: 4px;
+    background-color: $ui-01;
+    color: $text-01;
+    font-size: 85%;
+    cursor: pointer;
+
+    &:hover {
+      background-color: $ui-03;
+    }
+
+    &:focus {
+      outline: none;
+      border: 2px solid $brand-01;
+    }
+
+    &:active {
+      background: $copy-active;
+    }
+  }
+
+  .#{$prefix}--snippet--inline code {
+    padding: 0 $spacing-xs;
+  }
+
+  .#{$prefix}--snippet--inline.#{$prefix}--snippet--light {
+    background-color: $field-02;
+
+    &:hover {
+      background-color: rgba($brand-02, 0.1);
+    }
+  }
+
+  // Single Line Snippet
+  .#{$prefix}--snippet--single {
+    @include bx--snippet;
+    border: none;
+    max-width: rem(760px);
+    min-width: rem(320px);
+    height: rem(32px);
+    padding: 0 $spacing-3xl 0 $spacing-md;
+  }
+
+  .#{$prefix}--snippet--single .#{$prefix}--snippet-container {
+    display: flex;
+    align-items: center;
+    height: rem(32px);
+    overflow-x: scroll;
+    position: relative;
+    padding: 0 0 16px 0;
+    height: rem(48px);
+  }
+
+  .#{$prefix}--snippet--single pre {
+    white-space: nowrap;
+  }
+
+  // Multi Line Snippet
+  .#{$prefix}--snippet--multi {
+    @include bx--snippet;
+    border: none;
+    padding: $spacing-md;
+    min-width: rem(320px);
+    max-width: 100%;
+  }
+
+  .#{$prefix}--snippet--multi .#{$prefix}--snippet-container {
+    overflow: hidden;
+    position: relative;
+    max-height: rem(238px);
+    min-height: rem(56px);
+    transition: all $transition--base $carbon--standard-easing;
+  }
+
+  .#{$prefix}--snippet--expand.#{$prefix}--snippet--multi .#{$prefix}--snippet-container {
+    max-height: rem(1500px);
+  }
+
+  .#{$prefix}--snippet--multi .#{$prefix}--snippet-container pre {
+    white-space: pre-wrap;
+  }
+
+  //Copy Button
+  .#{$prefix}--snippet__icon {
+    fill: $text-02;
+    height: rem(16px);
+    width: rem(16px);
+    transition: all $transition--base $carbon--standard-easing;
+  }
+
+  .#{$prefix}--snippet-button {
+    @include reset;
+    cursor: pointer;
+    position: absolute;
+    top: 0;
+    right: 0;
+    border: none;
+    background-color: transparent;
+    outline: none;
+    padding: 0;
+    height: rem(32px);
+    width: rem(32px);
+    overflow: visible;
+
+    &:focus {
+      @include focus-outline('outline');
+    }
+  }
+
+  .#{$prefix}--snippet-button:hover {
+    background: $hover-field;
+  }
+
+  .#{$prefix}--snippet-button:active {
+    background: $copy-active;
+  }
+
+  .#{$prefix}--btn--copy__feedback {
+    @include font-family;
+    z-index: z('overlay');
+    font-weight: 400;
+    left: inherit;
+    top: 0.75rem;
+    right: 1.25rem;
+  }
+
+  .#{$prefix}--btn--copy__feedback:before,
+  .#{$prefix}--btn--copy__feedback:after {
+    background: #3d3d3d;
```

Comment:
Change color to the color token. 

Its `$ibm-colors__gray--80` but you might need to add a token for the snippet in theme

Reply:
Shoot, I totally meant to change that. Thanks for catching that @aledavila !